template/doc-frame.html

```diff
@@ -50,31 +50,26 @@
                                 <ul>
                                     {aside}
                                     <li class="laravel">
-                                        <h2>Laravel Basics</h2>
+                                        <h2>Laravel<img style="margin-left:4px;" src="img/external-link.svg" /></h2>
```

Comment:
Nice! But I would make this caption the link to the entry point of the external Laravel documentation and remove all of the sub-items.

There are two things which bother me:

 - The list of sub-items soon gets outdated and we have to keep it in sync with upstream. I don't see any value in making this effort 
 - The list is not complete. For example, when I started coding for Lychee, I needed totally different information about the Laravel framework than provided by the linked topics. If we curate the list of topics, then we should also be able to answer the question, why we chose exactly those topics and no other ones. (At least we should answer this question for ourselves.) For example, do we only expect future contributors to provide enhancement to the database but not to authentication? I doubt it, but then the question is "why this elective list"? That is why I suggested to only link to the framework on the top level, i.e. we say "Lychee uses Laravel as a framework, if you want to know more about Laravel, go here", but not provide a hand-picked list.